src/Oembed.elm

```diff
@@ -95,6 +93,14 @@ view customProviders options resourceUrl =
         |> Maybe.map (urlToIframe options resourceUrl)
 
 
+{-| Check if the passed in url matches any provider in [the hardcoded list](https://oembed.com/#section7)
+of oembed provider schemes.
```

Comment:
Looks good, could you also add a usage example in the doc comment here?

Reply:
Oh, sorry I meant a code snippet to demonstrate its usage, not an explanation of the usage. You can keep a little context about the usage, but could you add a code snippet here? It just makes it a little more concrete for users to read the example in the docs.

Reply:
You could use the test case from your unit test.